upcloud/testdata/upcloud_managed_object_storage/managed_object_storage_user_s2.tf

```
@@ -0,0 +1,27 @@
+variable "prefix" {
+  default = "tf-acc-test-objstov2-"
+  type    = string
+}
+
+variable "region" {
+  default = "europe-1"
+  type    = string
+}
+
+resource "upcloud_managed_object_storage" "user" {
+  name              = "${var.prefix}user"
```

```suggestion
  name              = "${var.prefix}objsto"
```
or not sure which postfix we have used for managed object storages elsewhere

---

Nevermind, I guess the user here refers to the testcase, not resource type 🙈